parser/src/parser/tests/random_values.rs

```diff
@@ -0,0 +1,47 @@
+use super::{build_parse_test, Identifier, RandBinding, RandomValues, Source, SourceSection::*};
+
+// RANDOM VALUES
+// ================================================================================================
+
+#[test]
+fn random_values_fixed_list_default_name() {
+    let source = "
+    random_values:
+        rand: [15]";
+    let expected = Source(vec![RandomValues(RandomValues::new(
+        Identifier("rand".to_string()),
+        15,
+        vec![],
+    ))]);
+    build_parse_test!(source).expect_ast(expected);
+}
+
+#[test]
+fn random_values_ident_vector_custom_name() {
+    let source = "
+    random_values:
+        alphas: [a, b[12], c]";
+    let expected = Source(vec![RandomValues(RandomValues::new(
+        Identifier("alphas".to_string()),
+        14,
+        vec![
+            RandBinding::new(Identifier("a".to_string()), 1),
+            RandBinding::new(Identifier("b".to_string()), 12),
+            RandBinding::new(Identifier("c".to_string()), 1),
+        ],
+    ))]);
+    build_parse_test!(source).expect_ast(expected);
+}
+
+#[test]
+fn random_values_empty_list() {
+    let source = "
+    random_values:
+        rand: []";
+    let expected = Source(vec![RandomValues(RandomValues::new(
+        Identifier("rand".to_string()),
+        0,
+        vec![],
+    ))]);
+    build_parse_test!(source).expect_ast(expected);
+}
```

Comment:
This test is good, but we should probably add a few error cases as well. (e.g. empty `random_values` section or more than one set of random values being declared)